CustomItems/Items/Scp1499.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections.Generic;
+using CustomItems.API;
+using Exiled.API.Features;
+using Exiled.Events.EventArgs;
+using MEC;
+using UnityEngine;
+
+namespace CustomItems.Items
+{
+    public class Scp1499 : CustomItem
+    {
+        public Scp1499(ItemType type, int itemId) : base(type, itemId)
+        {
+        }
+
+        public override string Name { get; set; } = Plugin.Singleton.Config.ItemConfigs.Scp1499Cfg.Name;
+        public override Dictionary<SpawnLocation, float> SpawnLocations { get; set; } = Plugin.Singleton.Config.ItemConfigs.Scp1499Cfg.SpawnLocations;
+        public override string Description { get; set; } = Plugin.Singleton.Config.ItemConfigs.Scp1499Cfg.Description;
+        public override int SpawnLimit { get; set; } = Plugin.Singleton.Config.ItemConfigs.Scp1499Cfg.SpawnLimit;
+
+
+        private Dictionary<Player, Vector3> scp1499Players = new Dictionary<Player, Vector3>();
+
+
+        protected override void LoadEvents()
+        {
+            Exiled.Events.Handlers.Server.WaitingForPlayers += OnWaitingForPlayers1499;
+            Exiled.Events.Handlers.Player.MedicalItemUsed += OnUsedMedicalItem;
+            Exiled.Events.Handlers.Player.DroppingItem += OnItemDropping;
+            base.LoadEvents();
+        }
+
+        protected override void UnloadEvents()
+        {
+            Exiled.Events.Handlers.Server.WaitingForPlayers -= OnWaitingForPlayers1499;
+            Exiled.Events.Handlers.Player.MedicalItemUsed -= OnUsedMedicalItem;
+            Exiled.Events.Handlers.Player.DroppingItem -= OnItemDropping;
+            base.UnloadEvents();
+        }
+
+
+        private void OnWaitingForPlayers1499()
+        {
+            scp1499Players.Clear();
+        }
+
+        private void OnUsedMedicalItem(UsedMedicalItemEventArgs ev)
+        {
+            if(CheckItem(ev.Player.CurrentItem))
+            {
+                scp1499Players.Add(ev.Player, ev.Player.Position);
+
+                ev.Player.Position = new UnityEngine.Vector3(152.93f, 978.03f, 93.64f);

```

Comment:
Please save this vector as a variable and name it appropriately. This qualifies as a "magic number", it does something, clearly of meaning, but I have no way by looking at the code of determining where on the map this vector is.